luacom/tLuaCOMTypeHandler.cpp

```
@@ -1913,34 +2382,25 @@ TYPEDESC tLuaCOMTypeHandler::processAliases(ITypeInfo* typeinfo,
-    userdef->Release();
     TYPECONV_ERROR("Could not understand user-defined type");
-  }
 
   TYPEDESC newtdesc;
   newtdesc.vt = 0;
-  if(typeattr->typekind == TKIND_ALIAS)
-  {
-    newtdesc = typeattr->tdescAlias;
-    newtdesc = processAliases(typeinfo, newtdesc);
-  }
+  if(typeattr.value->typekind == TKIND_ALIAS)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)